egs/wsj/s5/utils/prepare_subword_text.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
this shouldn't be at the top-level of utils/.  I think it, and the prepare_lang script, should be in utils/subword/.  The top level is getting too crowded.